sdv/dapr/client.py

```diff
@@ -64,6 +65,12 @@ async def wait_for_sidecar() -> None:
                 logger.debug(
                     "dapr: Health endpoint returned status code: %s", response.status
                 )
+            except HTTPError as error:
+                if error.code == HTTPStatus.INTERNAL_SERVER_ERROR:
+                    logger.info("Waiting for dapr sidecar...")
+                else:
+                    logger.error("Unexpected error from dapr sidecar: %d", error.code)
+                await asyncio.sleep(0.1)
             except BaseException as error:
                 logger.debug("%s", str(error))
```

Comment:
Shouldn't that be also logged on error level (or at least warning)?

Reply:
Waiting is expected since the sidecar comes up as separate process and takes a while to boot up. It is unfortunate that urllib raises an exception in case of code 500 and we have to catch it here.

Reply:
Yes, I know. But is that BaseException always just thrown while waiting makes sense - I mean it's a **Base**Exception.

Reply:
Oh you are referring to line 68, I agree it should be on error level, will change it.